File: c/c99/1094_experiencias.c
#include <stdio.h>

int main() {
    int N, quantia, total = 0, coelhos = 0, ratos = 0, sapos = 0;
    float percentual_c = 0, percentual_r = 0, percentual_s = 0;
    char tipo[50];
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        scanf("%d %s", &quantia, &tipo[50]);
        total += quantia;
        if (tipo[50] == "C") {
            coelhos += quantia;
        } else if (tipo[50] == "R") {
            ratos += quantia;
        } else if (tipo[50] == "S") {
            sapos += quantia;
        }
    }

    percentual_c = (coelhos*100)/total;
    percentual_r = (ratos*100)/total;
    percentual_s = (sapos*100)/total;

    printf("Total: %d cobaias\nTotal de coelhos: %d\nTotal de ratos: %d\nTotal de sapos: %d\nPercentual de coelhos: %.2f %%\nPercentual de ratos: %.2f %%\nPercentual de sapos: %.2f %%\n", total, coelhos, ratos, sapos, percentual_c, percentual_r, percentual_s);

    return 0;
}

File: python/1 - Iniciante/1094_experiencias.py
N = int(input())

Total = 0
coelhos = 0
ratos = 0
sapos = 0
percentual_c = percentual_r = percentual_s = 0

for num in range(1, N+1):
    Quantia, Tipo = input().split()
    Quantia = int(Quantia)
    Total += Quantia
    if Tipo == 'C':
        coelhos += Quantia
    elif Tipo == 'R':
        ratos += Quantia
    elif Tipo == 'S':
        sapos += Quantia

percentual_c = (coelhos*100)/Total
percentual_r = (ratos*100)/Total
percentual_s = (sapos*100)/Total

print(f'Total: {Total} cobaias')
print(f'Total de coelhos: {coelhos}')
print(f'Total de ratos: {ratos}')
print(f'Total de sapos: {sapos}')
print(f'Percentual de coelhos: {percentual_c:.2f} %')
print(f'Percentual de ratos: {percentual_r:.2f} %')
print(f'Percentual de sapos: {percentual_s:.2f} %')
